docker/docker-compose.yml

```diff
@@ -0,0 +1,170 @@
+version: '3.8'
+
+# Custom top-level network
+networks:
+  internal:
+  db:
+  web:
+
+services:
+  neo4j:
+    image: ${NEO4J_DOCKER_IMAGE:-neo4j:4.4}
+    hostname: neo4j
+    # Service-level network, which specifies the networks, from the list of the top-level networks (in this case only neo4j-internal), that the server will connect to.
+    # Adds a network alias (used in neo4j.conf when configuring the discovery members)
+    networks:
+      db:
+        aliases:
+          - neo4j-network
+    # The ports that will be accessible from outside the container - HTTP (7474) and Bolt (7687).
+    expose:
+      - 7687
+    # Uncomment the volumes to be mounted to make them accessible from outside the container.
+    volumes:
+      #- ./neo4j.conf:/conf/neo4j.conf # This is the main configuration file.
+      - ${NEO4J_DATA_STORE:?err}:/var/lib/neo4j/data
+      #- ./data/server:/var/lib/neo4j/data
+      #- ./logs/neo4j:/var/lib/neo4j/logs
+      #- ./conf/neo4j:/var/lib/neo4j/conf
+      #- ./import/neo4j:/var/lib/neo4j/import
+      #- ./metrics/neo4j:/var/lib/neo4j/metrics
+      #- ./licenses/neo4j:/var/lib/neo4j/licenses
+      #- ./ssl/neo4j:/var/lib/neo4j/ssl
+    # Passes the following environment variables to the container
+    environment:
+      - NEO4J_AUTH=${NEO4J_USER:?err}/${NEO4J_PASS:?err}
+      - NEO4J_EDITION=docker_compose
```

Comment:
What does this one do?

Reply:
No idea :upside_down_face: Just copying the docs